src/sap/fhir/model/r4/FHIRListBinding.js

```diff
@@ -162,7 +162,7 @@ sap.ui.define([
 				this.bInitial = false;
 				var oBindingInfo = this.oModel.getBindingInfo(this.sPath, this.oContext, this.bUnique);
 				var oResource = this.oModel.getProperty(oBindingInfo.getResourcePath()) || {};
-			    oResource.resourceType = oResource.resourceType || oBindingInfo.getResourceType(); 
+			    oResource.resourceType = oResource.resourceType || oBindingInfo.getResourceType();
```

Comment:
missing indent, isn't it?

Reply:
done.. eslint watch fix doesnot work :( neither does eslint --fix .. need to see